source/reference/logging.txt

```diff
@@ -68,6 +69,9 @@ For example:
 The currently active log file is ``mongosync.log``.  The other files are older
 logs that ``mongosync`` has rotated.
 
+.. note:: 
+
+   .. include:: /includes/fact-log-rotation-usr1-signal 
```

Comment:
[Optional suggestion]

I think we should try to avoid admonitions when possible so we don't end up over-using them. In this case, it might be reasonable to put this content into an h3 section called "Rotate Log File" or similar:

```
Rotate Log File
\``````````````

.. include:: /includes/fact-log-rotation-usr1-signal